_data/communication.yml

```diff
@@ -162,7 +162,8 @@ websites:
       sms: Yes
       software: Yes
       email: Yes
-      doc: https://help.salesforce.com/HTViewHelpDoc?id=security_require_two_factor_authentication.htm
+      hardware: Yes
+      doc: https://help.salesforce.com/articleView?id=security_overview_2fa.htm&type=5
```

Comment:
Is there a better link for end-users rather than admins? Perhaps https://trust.salesforce.com/en/security/2fa/ ?

Reply:
Good point. My only problem with the article you linked is that it seems to only promote their proprietary application, rather than an authenticator app in general. Thoughts on that?

Reply:
The only other relevant link I can find is a developer page, but that is even more cryptic. A bunch of the others just redirect to the admin page.

Reply:
I personally don't have too much of a problem with them promoting it. People who know the difference will be able to figure it out on their own. Otherwise, it should simplify support issues for Salesforce.

Reply:
Hi @mxxcon I think I found a good one: https://developer.salesforce.com/docs/atlas.en-us.identityImplGuide.meta/identityImplGuide/add_time-based_token.htm